bundles/remote_services/topology_manager/src/topology_manager.c

```diff
@@ -207,68 +182,62 @@ celix_status_t topologyManager_rsaAdded(void * handle, service_reference_pt unus
 	remote_service_admin_service_t *rsa = (remote_service_admin_service_t *) service;
 	celix_logHelper_log(manager->loghelper, CELIX_LOG_LEVEL_INFO, "TOPOLOGY_MANAGER: Added RSA");
 
-	celixThreadMutex_lock(&manager->rsaListLock);
+	celixThreadMutex_lock(&manager->managerLock);
 	arrayList_add(manager->rsaList, rsa);
-    celixThreadMutex_unlock(&manager->rsaListLock);
 
-	// add already imported services to new rsa
-    celixThreadMutex_lock(&manager->importedServicesLock);
-    hash_map_iterator_pt importedServicesIterator = hashMapIterator_create(manager->importedServices);
+	hash_map_iterator_pt importedServicesIterator = hashMapIterator_create(manager->importedServices);
 
-    while (hashMapIterator_hasNext(importedServicesIterator)) {
-        hash_map_entry_pt entry = hashMapIterator_nextEntry(importedServicesIterator);
-        endpoint_description_t *endpoint = hashMapEntry_getKey(entry);
-        if (scope_allowImport(manager->scope, endpoint)) {
-            import_registration_t *import = NULL;
-            celix_status_t status = rsa->importService(rsa->admin, endpoint, &import);
+	while (hashMapIterator_hasNext(importedServicesIterator)) {
+		hash_map_entry_pt entry = hashMapIterator_nextEntry(importedServicesIterator);
```

Comment:
indentation changed, tabs vs spaces?

Reply:
Oh, should be spaces. I'll see if I can fix this.

Reply:
This file is a mess: mostly it's tabs, but sometimes it's space. Probably the entire file should be reformatted in another PR.